### PR TITLE
depends: cctools 986, ld64 711 & libtapi 1300.0.6.5

### PR DESCRIPTION
**Pull request by fanquake**
Update [libtapi to `1300.0.6.5`](https://github.com/tpoechtrager/apple-libtapi/tree/1300.6.5), [cctools to `986` & ld64 to `711`](https://github.com/tpoechtrager/cctools-port/commits/986-ld64-711).

Updating these dependencies fixes the macOS build failures in #27897, when using a native GCC 12 toolchain, which means after this change, we can use use GCC 12 for all HOSTS, rather than splitting between 12 & 10 (for macOS).

Guix Build (x86_64):
```bash
a684a3e7668250610d653e5160115fb50b477339423335b185bd4716c8420e14  guix-build-7d5815293ed8/output/aarch64-linux-gnu/SHA256SUMS.part
9c66b82e2c5b6dbdcabdd962481485f71af5770e8bb573eaefb55d45e85e2875  guix-build-7d5815293ed8/output/aarch64-linux-gnu/bitcoin-7d5815293ed8-aarch64-linux-gnu-debug.tar.gz
63f82714a93e7a3c0e922cdcce7d3b27789d63cdf136dc3a71b74d61b0910ce8  guix-build-7d5815293ed8/output/aarch64-linux-gnu/bitcoin-7d5815293ed8-aarch64-linux-gnu.tar.gz
dcc79295462e0f585753959247ec02b273f5a933fadc5d528c266d56bbdaaf44  guix-build-7d5815293ed8/output/arm-linux-gnueabihf/SHA256SUMS.part
044fc0ad2a8615dd004579b5003fa981ebc4e3bfaacf5eba8a33cea02a9ba518  guix-build-7d5815293ed8/output/arm-linux-gnueabihf/bitcoin-7d5815293ed8-arm-linux-gnueabihf-debug.tar.gz
a032899be14fa2987dfd5a21865567d3beea703e778a2f91ec17eddb099d0c46  guix-build-7d5815293ed8/output/arm-linux-gnueabihf/bitcoin-7d5815293ed8-arm-linux-gnueabihf.tar.gz
27d43a97ea6c4ed2e84d9a9dd6c240e8e199174f4491ee1fb62abb1d950a96bd  guix-build-7d5815293ed8/output/arm64-apple-darwin/SHA256SUMS.part
029347186cc55c15152e8c255493ed457e58127ffa536982f3a632fc416b6662  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin-unsigned.dmg
791399447c33f7d3ed4d77fd22fde69aa7c87adda9291fca9508237b69b05b80  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin-unsigned.tar.gz
b9c72b7f37068889a83384598b6de97c8e39e7a097b661152084a5e69236541d  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin.tar.gz
6a249cc6d02d7398b9ed70b251422bf4bb1c2acdaafb5a885870e8ef20d43a89  guix-build-7d5815293ed8/output/dist-archive/bitcoin-7d5815293ed8.tar.gz
064e5e0df21aa177eaf8d67254ad5166b190559f69da68d34174a323d8c975c6  guix-build-7d5815293ed8/output/powerpc64-linux-gnu/SHA256SUMS.part
6aee00585bdd680d42f3341df523b7173dc06defa8f51f98af38c8518ddf4532  guix-build-7d5815293ed8/output/powerpc64-linux-gnu/bitcoin-7d5815293ed8-powerpc64-linux-gnu-debug.tar.gz
d6e51b521c84276d5c9606a96ee36b94bc235370064cf75148d15d3a4011ed9d  guix-build-7d5815293ed8/output/powerpc64-linux-gnu/bitcoin-7d5815293ed8-powerpc64-linux-gnu.tar.gz
e0a133afe0e0e7894d3c6110b7b1895efff8f45abbe473d95100d3c5615dc855  guix-build-7d5815293ed8/output/powerpc64le-linux-gnu/SHA256SUMS.part
8d62884b81fca76720284ae4d5b2a0d0ebfcbcf533d9936c8aa8ab04739540e9  guix-build-7d5815293ed8/output/powerpc64le-linux-gnu/bitcoin-7d5815293ed8-powerpc64le-linux-gnu-debug.tar.gz
c4c90dd3c83a84aa074ab5dc77b82fd2a26d872547037ef2a8a2d11abba1db98  guix-build-7d5815293ed8/output/powerpc64le-linux-gnu/bitcoin-7d5815293ed8-powerpc64le-linux-gnu.tar.gz
dd28d196e89f74cf7666b347e1cbdd16eb90bfe0d5b054ac2edbf0e4f755bd9a  guix-build-7d5815293ed8/output/riscv64-linux-gnu/SHA256SUMS.part
8d3ace8a17d4c1a13786adb3669b1d929485b164c59efb1fe7cf8fe3aad90f73  guix-build-7d5815293ed8/output/riscv64-linux-gnu/bitcoin-7d5815293ed8-riscv64-linux-gnu-debug.tar.gz
bf530b7944dfdd9ffd03b2be541fa349e8598cb39ee5e1bd4be0b5b48bdfcc31  guix-build-7d5815293ed8/output/riscv64-linux-gnu/bitcoin-7d5815293ed8-riscv64-linux-gnu.tar.gz
6172224f94d52eb22a3749b1e3c0639476dfdaff47d15c6a8ec3e6dbf8763f9b  guix-build-7d5815293ed8/output/x86_64-apple-darwin/SHA256SUMS.part
f59f68ad4783ff1ecb09cd84f15bbfc803c856971de5a131fbb1777cf8f1b79a  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin-unsigned.dmg
ac31df20d1e5f3a73aadb5455e804af053b179ca6471b5a57afe3c7df0f09128  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin-unsigned.tar.gz
a33f661386d415f621ab17edf7fe3efa61f0c2b0c3003fecd4f5f6898129992d  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin.tar.gz
03104057b86de994258587ff5a8de16f48354a7493527e5bb2559714c0e2bf39  guix-build-7d5815293ed8/output/x86_64-linux-gnu/SHA256SUMS.part
a9c3f233629038978b9fa11c1353b4e686406851c0e84a5539cf3bc90722d98b  guix-build-7d5815293ed8/output/x86_64-linux-gnu/bitcoin-7d5815293ed8-x86_64-linux-gnu-debug.tar.gz
61db7ec4aa306a6f79628b217c05387b69002d123fcf07d6784cce03d9a8574d  guix-build-7d5815293ed8/output/x86_64-linux-gnu/bitcoin-7d5815293ed8-x86_64-linux-gnu.tar.gz
7f5b42d02d8c026492376b644a94a45613155a0ccc1d3ef9f067b3370d45c4d2  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/SHA256SUMS.part
de23de0fb32c77f9ff91ffe554347c7ddc77fae22212d0e609389b4b7a1d6e29  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64-debug.zip
2131ce905aa3452883a9f47b6618506a6cb90c2a1725818084bc4e08856ff8d3  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64-setup-unsigned.exe
5555d28d9ea14a50ab296c2b1f1db7f7acde933009334999b6be29a2ed469bc0  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64-unsigned.tar.gz
7dc29c317c140ae8a8f487d2d66761c3fae90a5521dd4bfd0d071adbfafb30ef  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64.zip
```

### CODE REVIEW

**Comment by tobtoht** [comment]
Guix builds:

```
a684a3e7668250610d653e5160115fb50b477339423335b185bd4716c8420e14  guix-build-7d5815293ed8/output/aarch64-linux-gnu/SHA256SUMS.part
9c66b82e2c5b6dbdcabdd962481485f71af5770e8bb573eaefb55d45e85e2875  guix-build-7d5815293ed8/output/aarch64-linux-gnu/bitcoin-7d5815293ed8-aarch64-linux-gnu-debug.tar.gz
63f82714a93e7a3c0e922cdcce7d3b27789d63cdf136dc3a71b74d61b0910ce8  guix-build-7d5815293ed8/output/aarch64-linux-gnu/bitcoin-7d5815293ed8-aarch64-linux-gnu.tar.gz
dcc79295462e0f585753959247ec02b273f5a933fadc5d528c266d56bbdaaf44  guix-build-7d5815293ed8/output/arm-linux-gnueabihf/SHA256SUMS.part
044fc0ad2a8615dd004579b5003fa981ebc4e3bfaacf5eba8a33cea02a9ba518  guix-build-7d5815293ed8/output/arm-linux-gnueabihf/bitcoin-7d5815293ed8-arm-linux-gnueabihf-debug.tar.gz
a032899be14fa2987dfd5a21865567d3beea703e778a2f91ec17eddb099d0c46  guix-build-7d5815293ed8/output/arm-linux-gnueabihf/bitcoin-7d5815293ed8-arm-linux-gnueabihf.tar.gz
27d43a97ea6c4ed2e84d9a9dd6c240e8e199174f4491ee1fb62abb1d950a96bd  guix-build-7d5815293ed8/output/arm64-apple-darwin/SHA256SUMS.part
029347186cc55c15152e8c255493ed457e58127ffa536982f3a632fc416b6662  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin-unsigned.dmg
791399447c33f7d3ed4d77fd22fde69aa7c87adda9291fca9508237b69b05b80  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin-unsigned.tar.gz
b9c72b7f37068889a83384598b6de97c8e39e7a097b661152084a5e69236541d  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin.tar.gz
6a249cc6d02d7398b9ed70b251422bf4bb1c2acdaafb5a885870e8ef20d43a89  guix-build-7d5815293ed8/output/dist-archive/bitcoin-7d5815293ed8.tar.gz
064e5e0df21aa177eaf8d67254ad5166b190559f69da68d34174a323d8c975c6  guix-build-7d5815293ed8/output/powerpc64-linux-gnu/SHA256SUMS.part
6aee00585bdd680d42f3341df523b7173dc06defa8f51f98af38c8518ddf4532  guix-build-7d5815293ed8/output/powerpc64-linux-gnu/bitcoin-7d5815293ed8-powerpc64-linux-gnu-debug.tar.gz
d6e51b521c84276d5c9606a96ee36b94bc235370064cf75148d15d3a4011ed9d  guix-build-7d5815293ed8/output/powerpc64-linux-gnu/bitcoin-7d5815293ed8-powerpc64-linux-gnu.tar.gz
e0a133afe0e0e7894d3c6110b7b1895efff8f45abbe473d95100d3c5615dc855  guix-build-7d5815293ed8/output/powerpc64le-linux-gnu/SHA256SUMS.part
8d62884b81fca76720284ae4d5b2a0d0ebfcbcf533d9936c8aa8ab04739540e9  guix-build-7d5815293ed8/output/powerpc64le-linux-gnu/bitcoin-7d5815293ed8-powerpc64le-linux-gnu-debug.tar.gz
c4c90dd3c83a84aa074ab5dc77b82fd2a26d872547037ef2a8a2d11abba1db98  guix-build-7d5815293ed8/output/powerpc64le-linux-gnu/bitcoin-7d5815293ed8-powerpc64le-linux-gnu.tar.gz
dd28d196e89f74cf7666b347e1cbdd16eb90bfe0d5b054ac2edbf0e4f755bd9a  guix-build-7d5815293ed8/output/riscv64-linux-gnu/SHA256SUMS.part
8d3ace8a17d4c1a13786adb3669b1d929485b164c59efb1fe7cf8fe3aad90f73  guix-build-7d5815293ed8/output/riscv64-linux-gnu/bitcoin-7d5815293ed8-riscv64-linux-gnu-debug.tar.gz
bf530b7944dfdd9ffd03b2be541fa349e8598cb39ee5e1bd4be0b5b48bdfcc31  guix-build-7d5815293ed8/output/riscv64-linux-gnu/bitcoin-7d5815293ed8-riscv64-linux-gnu.tar.gz
6172224f94d52eb22a3749b1e3c0639476dfdaff47d15c6a8ec3e6dbf8763f9b  guix-build-7d5815293ed8/output/x86_64-apple-darwin/SHA256SUMS.part
f59f68ad4783ff1ecb09cd84f15bbfc803c856971de5a131fbb1777cf8f1b79a  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin-unsigned.dmg
ac31df20d1e5f3a73aadb5455e804af053b179ca6471b5a57afe3c7df0f09128  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin-unsigned.tar.gz
a33f661386d415f621ab17edf7fe3efa61f0c2b0c3003fecd4f5f6898129992d  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin.tar.gz
03104057b86de994258587ff5a8de16f48354a7493527e5bb2559714c0e2bf39  guix-build-7d5815293ed8/output/x86_64-linux-gnu/SHA256SUMS.part
a9c3f233629038978b9fa11c1353b4e686406851c0e84a5539cf3bc90722d98b  guix-build-7d5815293ed8/output/x86_64-linux-gnu/bitcoin-7d5815293ed8-x86_64-linux-gnu-debug.tar.gz
61db7ec4aa306a6f79628b217c05387b69002d123fcf07d6784cce03d9a8574d  guix-build-7d5815293ed8/output/x86_64-linux-gnu/bitcoin-7d5815293ed8-x86_64-linux-gnu.tar.gz
7f5b42d02d8c026492376b644a94a45613155a0ccc1d3ef9f067b3370d45c4d2  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/SHA256SUMS.part
de23de0fb32c77f9ff91ffe554347c7ddc77fae22212d0e609389b4b7a1d6e29  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64-debug.zip
2131ce905aa3452883a9f47b6618506a6cb90c2a1725818084bc4e08856ff8d3  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64-setup-unsigned.exe
5555d28d9ea14a50ab296c2b1f1db7f7acde933009334999b6be29a2ed469bc0  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64-unsigned.tar.gz
7dc29c317c140ae8a8f487d2d66761c3fae90a5521dd4bfd0d071adbfafb30ef  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64.zip
```

**Comment by TheCharlatan** [comment]
Guix build (same on both aarch64 and x86_64 hosts):

```
a684a3e7668250610d653e5160115fb50b477339423335b185bd4716c8420e14  guix-build-7d5815293ed8/output/aarch64-linux-gnu/SHA256SUMS.part
9c66b82e2c5b6dbdcabdd962481485f71af5770e8bb573eaefb55d45e85e2875  guix-build-7d5815293ed8/output/aarch64-linux-gnu/bitcoin-7d5815293ed8-aarch64-linux-gnu-debug.tar.gz
63f82714a93e7a3c0e922cdcce7d3b27789d63cdf136dc3a71b74d61b0910ce8  guix-build-7d5815293ed8/output/aarch64-linux-gnu/bitcoin-7d5815293ed8-aarch64-linux-gnu.tar.gz
dcc79295462e0f585753959247ec02b273f5a933fadc5d528c266d56bbdaaf44  guix-build-7d5815293ed8/output/arm-linux-gnueabihf/SHA256SUMS.part
044fc0ad2a8615dd004579b5003fa981ebc4e3bfaacf5eba8a33cea02a9ba518  guix-build-7d5815293ed8/output/arm-linux-gnueabihf/bitcoin-7d5815293ed8-arm-linux-gnueabihf-debug.tar.gz
a032899be14fa2987dfd5a21865567d3beea703e778a2f91ec17eddb099d0c46  guix-build-7d5815293ed8/output/arm-linux-gnueabihf/bitcoin-7d5815293ed8-arm-linux-gnueabihf.tar.gz
27d43a97ea6c4ed2e84d9a9dd6c240e8e199174f4491ee1fb62abb1d950a96bd  guix-build-7d5815293ed8/output/arm64-apple-darwin/SHA256SUMS.part
029347186cc55c15152e8c255493ed457e58127ffa536982f3a632fc416b6662  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin-unsigned.dmg
791399447c33f7d3ed4d77fd22fde69aa7c87adda9291fca9508237b69b05b80  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin-unsigned.tar.gz
b9c72b7f37068889a83384598b6de97c8e39e7a097b661152084a5e69236541d  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin.tar.gz
6a249cc6d02d7398b9ed70b251422bf4bb1c2acdaafb5a885870e8ef20d43a89  guix-build-7d5815293ed8/output/dist-archive/bitcoin-7d5815293ed8.tar.gz
064e5e0df21aa177eaf8d67254ad5166b190559f69da68d34174a323d8c975c6  guix-build-7d5815293ed8/output/powerpc64-linux-gnu/SHA256SUMS.part
6aee00585bdd680d42f3341df523b7173dc06defa8f51f98af38c8518ddf4532  guix-build-7d5815293ed8/output/powerpc64-linux-gnu/bitcoin-7d5815293ed8-powerpc64-linux-gnu-debug.tar.gz
d6e51b521c84276d5c9606a96ee36b94bc235370064cf75148d15d3a4011ed9d  guix-build-7d5815293ed8/output/powerpc64-linux-gnu/bitcoin-7d5815293ed8-powerpc64-linux-gnu.tar.gz
e0a133afe0e0e7894d3c6110b7b1895efff8f45abbe473d95100d3c5615dc855  guix-build-7d5815293ed8/output/powerpc64le-linux-gnu/SHA256SUMS.part
8d62884b81fca76720284ae4d5b2a0d0ebfcbcf533d9936c8aa8ab04739540e9  guix-build-7d5815293ed8/output/powerpc64le-linux-gnu/bitcoin-7d5815293ed8-powerpc64le-linux-gnu-debug.tar.gz
c4c90dd3c83a84aa074ab5dc77b82fd2a26d872547037ef2a8a2d11abba1db98  guix-build-7d5815293ed8/output/powerpc64le-linux-gnu/bitcoin-7d5815293ed8-powerpc64le-linux-gnu.tar.gz
dd28d196e89f74cf7666b347e1cbdd16eb90bfe0d5b054ac2edbf0e4f755bd9a  guix-build-7d5815293ed8/output/riscv64-linux-gnu/SHA256SUMS.part
8d3ace8a17d4c1a13786adb3669b1d929485b164c59efb1fe7cf8fe3aad90f73  guix-build-7d5815293ed8/output/riscv64-linux-gnu/bitcoin-7d5815293ed8-riscv64-linux-gnu-debug.tar.gz
bf530b7944dfdd9ffd03b2be541fa349e8598cb39ee5e1bd4be0b5b48bdfcc31  guix-build-7d5815293ed8/output/riscv64-linux-gnu/bitcoin-7d5815293ed8-riscv64-linux-gnu.tar.gz
6172224f94d52eb22a3749b1e3c0639476dfdaff47d15c6a8ec3e6dbf8763f9b  guix-build-7d5815293ed8/output/x86_64-apple-darwin/SHA256SUMS.part
f59f68ad4783ff1ecb09cd84f15bbfc803c856971de5a131fbb1777cf8f1b79a  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin-unsigned.dmg
ac31df20d1e5f3a73aadb5455e804af053b179ca6471b5a57afe3c7df0f09128  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin-unsigned.tar.gz
a33f661386d415f621ab17edf7fe3efa61f0c2b0c3003fecd4f5f6898129992d  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin.tar.gz
03104057b86de994258587ff5a8de16f48354a7493527e5bb2559714c0e2bf39  guix-build-7d5815293ed8/output/x86_64-linux-gnu/SHA256SUMS.part
a9c3f233629038978b9fa11c1353b4e686406851c0e84a5539cf3bc90722d98b  guix-build-7d5815293ed8/output/x86_64-linux-gnu/bitcoin-7d5815293ed8-x86_64-linux-gnu-debug.tar.gz
61db7ec4aa306a6f79628b217c05387b69002d123fcf07d6784cce03d9a8574d  guix-build-7d5815293ed8/output/x86_64-linux-gnu/bitcoin-7d5815293ed8-x86_64-linux-gnu.tar.gz
7f5b42d02d8c026492376b644a94a45613155a0ccc1d3ef9f067b3370d45c4d2  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/SHA256SUMS.part
de23de0fb32c77f9ff91ffe554347c7ddc77fae22212d0e609389b4b7a1d6e29  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64-debug.zip
2131ce905aa3452883a9f47b6618506a6cb90c2a1725818084bc4e08856ff8d3  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64-setup-unsigned.exe
5555d28d9ea14a50ab296c2b1f1db7f7acde933009334999b6be29a2ed469bc0  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64-unsigned.tar.gz
7dc29c317c140ae8a8f487d2d66761c3fae90a5521dd4bfd0d071adbfafb30ef  guix-build-7d5815293ed8/output/x86_64-w64-mingw32/bitcoin-7d5815293ed8-win64.zip
```

**Review by sedited — APPROVED** [review]
ACK 7d5815293ed8a3dea68b61a78944e410f02b147f

Tested on macos 11.1

File: depends/packages/native_cctools.mk [hunk 1]
```diff
@@ -18,15 +17,9 @@ endef
 ifneq ($(strip $(FORCE_USE_SYSTEM_CLANG)),)
 define $(package)_preprocess_cmds
   mkdir -p $($(package)_staging_prefix_dir)/lib && \
-  cp $(llvm_lib_dir)/libLTO.so $($(package)_staging_prefix_dir)/lib/ && \
-  cp -f $(BASEDIR)/config.guess $(BASEDIR)/config.sub cctools && \
```
Comment (hebasto) [inline depends/packages/native_cctools.mk:22]: Why this line removed? I understand that it works now without it, but:
- there were related problems in the past, for example https://github.com/bitcoin/bitcoin/pull/22526
- this change is unrelated to the PR topic and omitted in the PR description

Comment (fanquake) [reply depends/packages/native_cctools.mk:22]: The new source has a newer `config.guess` and `config.sub`, which has the arm triple we need.

> there were related problems in the past,

Ok? However the triple isn't going to be removed in the future, so I'm not sure what you're point is.

> this change is unrelated to the PR topic and omitted in the PR description

How is it unrelated? Now that we are using newer sources, we don't need to copy over additional files.

Comment (fanquake) [reply depends/packages/native_cctools.mk:22]: Also, all of this code is going to be deleted in the near future in any case (#21778). The only reason it's being updated now is to avoid an awkward split in the GCC toolchains used in the Guix builds, otherwise, we'd avoid bumping it again.

**Review by hebasto — APPROVED** [review]
ACK 7d5815293ed8a3dea68b61a78944e410f02b147f

Guix builds:
```
x86_64
27d43a97ea6c4ed2e84d9a9dd6c240e8e199174f4491ee1fb62abb1d950a96bd  guix-build-7d5815293ed8/output/arm64-apple-darwin/SHA256SUMS.part
029347186cc55c15152e8c255493ed457e58127ffa536982f3a632fc416b6662  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin-unsigned.dmg
791399447c33f7d3ed4d77fd22fde69aa7c87adda9291fca9508237b69b05b80  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin-unsigned.tar.gz
b9c72b7f37068889a83384598b6de97c8e39e7a097b661152084a5e69236541d  guix-build-7d5815293ed8/output/arm64-apple-darwin/bitcoin-7d5815293ed8-arm64-apple-darwin.tar.gz
6a249cc6d02d7398b9ed70b251422bf4bb1c2acdaafb5a885870e8ef20d43a89  guix-build-7d5815293ed8/output/dist-archive/bitcoin-7d5815293ed8.tar.gz
6172224f94d52eb22a3749b1e3c0639476dfdaff47d15c6a8ec3e6dbf8763f9b  guix-build-7d5815293ed8/output/x86_64-apple-darwin/SHA256SUMS.part
f59f68ad4783ff1ecb09cd84f15bbfc803c856971de5a131fbb1777cf8f1b79a  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin-unsigned.dmg
ac31df20d1e5f3a73aadb5455e804af053b179ca6471b5a57afe3c7df0f09128  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin-unsigned.tar.gz
a33f661386d415f621ab17edf7fe3efa61f0c2b0c3003fecd4f5f6898129992d  guix-build-7d5815293ed8/output/x86_64-apple-darwin/bitcoin-7d5815293ed8-x86_64-apple-darwin.tar.gz
```
